RecoLocalTracker/SiStripClusterizer/python/SiStripClusterChargeCut_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 SiStripClusterChargeCutNone = cms.PSet(
-    value     = cms.double(-1.0)
+    value     = cms.double(0.0)
```

Comment:
Hi all,
I have a very vague recollection of the problem, but in general I would not use side-effects (the CCC) to fix local-reco problems. If clusters with 0 charge are a problem, we should get rid of them explicitly in local-reco, not as a side-effect of CCC (like you seem to imply here, since -1 meant no cut, while your proposal means 0 is always a valid cut)